### DISCUSSION TITLE
Using a VPN?

### DISCUSSION THREAD

**Discussion by kadisonm** [sboesen/remotely-sync#112] [General]
Is it possible to use this over a VPN? Is there any security implications with this. I don't have a strong understanding of it, but does the traffic going to the VPN server impact anything?

Also, I tried syncing while my VPN was on and it can't seem to fetch the metadata.

**Comment by kadisonm** [comment]
Actually this might not be my VPN. I'm finding it's taking a really long time to fetch the metadata on my computer and phone.

**Reply by sboesen** [reply]
VPN shouldn't affect this unless your VPN is either 1) blocking requests or 2) blocked by a cloud provider. Shouldn't be any security impact (apart from running a likely third-party VPN client on your device). 

How large is your metadata file? I think this grows over time, and probably even more when we start storing more metadata like mtime remotely.

**Reply by kadisonm** [reply]
I think it is a separate issue. I just thought it was related because I got a network changed error but that was probably because the VPN turned on halfway.

I'm not sure why but the plugins on my device take a very long time to fetch the metadata. With my PC not being able to sync at all and printing 'Metadata on remote True' message every few minutes.

**Reply by sboesen** [reply]
Hmm. I do wonder if the size of the metadata file is coming into play - assuming this only affects metadata sync and not other file sync (sounds like you can't sync at all?)

**Reply by kadisonm** [reply]
I moved this to a new issue #115 since I think you're right.
